cmd/operator/main.go

```diff
@@ -51,6 +53,16 @@ func main() {
 	}
 	config.Credentials = credentials.NewTLSCredentials(certChainPath)
 
+	go func() {
```

Comment:
Could we possibly take this common code to health package and export method StartHealthzServer(..)  but we need to create separate instance in that method..?

Reply:
No, we need to keep the server itself outside of the start scope, as it would need to be acted upon in the future to change the state. Regardless, this would over-encapsulation.

Reply:
sounds good.